backend/src/main/java/org/polsl/backend/repository/SoftwareRepository.java

```diff
@@ -5,6 +5,10 @@
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface SoftwareRepository extends CrudRepository<Software, Long> {
+
+//    List<Software> findAllBySoftwareIsNull();
```

Comment:
Nie wrzucajmy zakomentowanego kodu - ja Ci to tylko tymczasowo zakomentowałem na Twoim kompie xD

Reply:
Inny komentarz usunąłem, ten mi umknął 😅

Reply:
Spoko, zdarza się xD